cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs

```diff
@@ -293,6 +293,24 @@ impl pallet_assets_freezer::Config<AssetsFreezerInstance> for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 }
 
+parameter_types! {
+	pub const TrustBackedMinVestedTransfer: Balance = 100 * CENTS;
+}
+
+pub type TrustBackedAssetsVestingInstance = pallet_assets_vesting::Instance1;
+
+impl pallet_assets_vesting::Config<TrustBackedAssetsVestingInstance> for Runtime {
+	type ForceOrigin = EnsureRoot<AccountId>;
+	type Assets = Assets;
+	type Freezer = AssetsFreezer;
+	type BlockNumberToBalance = ConvertInto;
+	type RuntimeFreezeReason = RuntimeFreezeReason;
+	type WeightInfo = ();
```

Comment:
The CI added the weight, let's link them here.

Reply:
Had to revert the [link](https://github.com/paritytech/polkadot-sdk/pull/7404/commits/3b722e531647ab6d26cc00129d101a28df831a6e). Apparently these weights (here and in AH westend config) belong to another pallet (probably `pallet-assets`), but definitely not to `pallet_assets_vesting::weights::WeightInfo`